tests/test_beatmap.py

```diff
@@ -0,0 +1,108 @@
+from slider import Beatmap
+import slider.beatmap
+import slider.curve
+from slider.position import Position
+from datetime import timedelta
+from math import isclose
+
+tatoe_path = "data/AKINO from bless4 & CHiCO with " + \
+        "HoneyWorks - MIIRO vs. Ai no Scenario (monstrata) [Tatoe].osu"
+
+
+def test_beatmap_parameters():
+    beatmap = Beatmap.from_path(tatoe_path)
```

Comment:
this will depend on where the tests are run from. If you want to put this file in the sample_data dir, then we can open it with https://github.com/llllllllll/slider/blob/master/slider/example_data/beatmaps/__init__.py#L6

Reply:
testing only works from the test directory. It may be beneficial to merge sample_data and the data dir in tests dir but I think the data for tests should be independent from sample data. No big difference